Geometry_Engine/Create/ShapeProfile.cs

```diff
@@ -194,5 +481,19 @@ private static List<ICurve> MirrorAboutLocalZ(this List<ICurve> curves)
         }
 
         /***************************************************/
+
+        private static void InvalidRatioError(string inputs)
```

Comment:
Would consider just using the second method instead to keep things consistent, but not a big one.